app/Main.hs

```diff
@@ -199,16 +199,36 @@ updateImpl config@PackageConfig{ depends } = do
   echoT ("Updating " <> pack (show (length trans)) <> " packages...")
   forConcurrently_ trans . uncurry $ installOrUpdate (set config)
 
+pursCmd :: IO Text
+pursCmd = do
+  purs <- which "purs"
+  cmd <- which "purs.cmd"
```

Comment:
Where is `purs.cmd` from again?

Reply:
When you install an npm package globally using npm (or yarn, I suppose), a cmd file is created for each bin script the package has. The cmd file is executable and passes the bin script and any arguments to node. 

So, for example, purescript has bin/purs.js. So npm creates `purs.cmd` on the path. And `purs.cmd` runs `purs.js`; like this --> `node  "%~dp0\node_modules\purescript\bin\purs.js" %*`... well, there's a couple other steps, but that's basically it.

So windows users who use cmd.exe or powershell, instead of an msys terminal or git shell or something, need to be able find and execute the `purs.cmd` script if they are getting purescript from npm.

Reply:
I think I would rather leave `purs.cmd` out here, and encourage Windows users to either a) use the binaries directly, or b) temporarily modify their path to include wherever NPM puts them. I think it would reduce some complexity here, and I really don't think we should build in support for NPM. What do you think?

Reply:
I don't think that's a good idea - you're basically saying "we don't support the most common way of installing purescript" on Windows if `purs.cmd` is ignored.

Reply:
Honestly, I don't think psc-package should have support for building anyway now that Pulp has support for using psc-package. I'd rather delete the code than have custom code for each way PureScript might be installed.

Reply:
It seems like we're going to be breaking stuff again soon anyway (#67) so it's a good time to remove the `build` feature (which was always marked as temporary anyway).

Reply:
I'm with @garyb; Pulp has code to handle this specific case and it hasn't been a maintenance burden. Based on conversations I've had supporting Pulp I think there are definitely Windows users who, if they hit this, would struggle to identify the problem. They might even conclude that their PS install was broken, fiddle with it, and break things even more.

Reply:
I suppose we do need it for the version check though.

I know it seems like primitive obsession, or something, but it bothers me that there is a sensible default on all decent OSes (just put it on your PATH), but not on Windows, and we have to make special cases there, which then have to be documented.

Reply:
Oh if we're removing `build` altogether that works too :+1:

Reply:
I feel pretty alienated when project's tooling doesn't support windows native. In this case, the work around isn't that big of a deal, but I'm sure it will still drive people away.

Reply:
We could remove `build`, but then we'd still have `verify` which uses `purs`.

Reply:
Yeah we need it to check the compiler version too, actually, so removing it isn't really a solution.